src/Util/MIMETypeUtil.m

```diff
@@ -452,4 +453,1087 @@ + (NSString *)fileExtensionForMIMEType:(NSString *)mimeType
     return result;
 }
 
++ (NSDictionary *)genericMIMETypesToExtensionTypes
+{
+    static NSDictionary *result = nil;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        result = @{
```

Comment:
Can you leave a comment about the source of authority for this list in case there are errors/updates in the future?